openmetadata-ui/src/main/resources/ui/cypress/common/common.js

```diff
@@ -1123,3 +1123,32 @@ export const visitDatabaseSchemaDetailsPage = ({
     .contains(databaseSchemaName)
     .click();
 };
+
+const dataTransfer = new DataTransfer();
+
+export const dragAndDropElement = (dragElement, dropTarget, isHeader) => {
```

Comment:
Let's not add anything to common js
We can have DragAndDrop.ts instead in the utils